src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Insight.cs

```diff
@@ -6,10 +6,8 @@
 
 namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
 {
-    /// <summary>
-    /// Class representing Insight
-    /// </summary>
-    public class Insight
+
+    public class InsightBase
```

Comment:
this needs to abstract class

Reply:
I removed base class so that I wouldn't break old detectors